lib/vsphere.rb

```diff
@@ -266,7 +269,7 @@ def generate_clone_spec(source_config, dc, resource_pool, cpus, memory, ds_name,
     # CPU and RAM
     clone_spec.config.numCPUs  = Integer(cpus)
     clone_spec.config.memoryMB = Integer(memory)
-
+    clone_spec.config.nestedHVEnabled = !!virt
```

Comment:
Double bang is kind-of frowned upon.
Furthermore, you're kind of putting the logic deeper into code.

Since there's already a validate() function, at the end of it you should set the option to a boolean.

`options[:virt] ||= false`

Reply:
Per our conversation this makes sense since we are typecasting other variables in this stanza, and we need need to ensure this is a boolean